### PR TITLE
added note about DNS DLB Loadbalancing

### PR DESCRIPTION
**Pull request by nparente-mulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: cloudhub-1/modules/ROOT/pages/cloudhub-dedicated-load-balancer.adoc [hunk 1]
```diff
@@ -56,6 +56,7 @@ Switching between the two groups of static IP addresses enables zero-downtime re
 
 * Because static IPs change when you restart or update your DLB, use the DLB's DNS address rather than the IP addresses when calling the DLB from an external service. Ensure that the client includes the necessary retry mechanisms to account for the DNS switch.
 * To avoid connection issues when the load balancer is restarting, adhere to the DNS TTL (30 seconds), including connection keep alive. The connections are closed after the new workers are online and the TTL expires.
+* DLB uses DNS-based routing to distribute traffic across DLB workers and returns the IP address of all DLB workers to the calling client. A DLB worker might receive all the client traffic if a single client sends thousands of transactions per second within the TTL window. To avoid this, clients should distribute the traffic between the DLB workers by rotating between the resolved IP addresses.
```
Comment (nbedarkar) [inline cloudhub-1/modules/ROOT/pages/cloudhub-dedicated-load-balancer.adoc]: Should we reword this to show no matter how many requests it can go to a single worker?
DLB uses DNS-based routing to distribute traffic across DLB workers and returns the IP address of all DLB workers to the calling client. A DLB worker might receive all the client traffic within the TTL window even if a single client sends small or thousands of transactions per second. To avoid this, clients should distribute the traffic between the DLB workers by rotating between the resolved IP addresses.

**Review by nbedarkar — CHANGES_REQUESTED** [review]

**Review by nbedarkar — DISMISSED** [review]

**Review by tasmoore — APPROVED** [review]
